wfl/Database/MongoDBManager.cs

```diff
@@ -32,5 +32,32 @@ public MongoDBManager()
             IAsyncCursor<RestaurantDto> task = await collection.FindAsync(r => true, null);
             return task.ToEnumerable<RestaurantDto>();
         }
+
+        public async Task<IEnumerable<FoodOrderDto>> GetAllFoodOrdersAsync()
+        {
+            IMongoCollection<FoodOrderDto> collection = db.GetCollection<FoodOrderDto>("food-orders");
+            IAsyncCursor<FoodOrderDto> task = await collection.FindAsync(order => true, null);
+            return task.ToEnumerable<FoodOrderDto>();
+        }
+
+        public async Task InsertFoodOrderAsync(FoodOrderDto order)
+        {
+            order.Id = MongoDB.Bson.ObjectId.GenerateNewId().ToString();
```

Comment:
why are we generating a new id?  Can't we just let the database generate it like we did for restaurant?